### ISSUE TITLE
13 Pro Max iOS 15.7.2 Misaka 8.0.1/2/3/4/5/6/7/8，TIPS are all gray, unable to enter KFD mode and restart repeatedly

### ISSUE THREAD

**Issue by lxjinfo** [straight-tamago/misaka#150]
13 Pro Max iOS 15.7.2 Misaka 8.0.1/2/3/4/5/6/7/8，TIPS are all gray, unable to enter KFD mode and restart repeatedly

**Comment by qinghuaweitian** [comment]
Yes, I have the same problem. My device is air5 M1 15.7.2 TIPS. If you choose KFD mode in gray, the device will restart, but it cannot be changed to KFD mode.
